app/src/main/java/ch/hikemate/app/ui/components/BackButton.kt

```diff
@@ -0,0 +1,41 @@
+package ch.hikemate.app.ui.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import ch.hikemate.app.ui.navigation.NavigationActions
+
+const val BACK_BUTTON_TEST_TAG = "backButton"
```

Comment:
The reason behind this was that the `BackButton` will be used in the whole project. As the navigation test tags, It is global for better usability
```kotlin
const val TEST_TAG_SIDEBAR_BUTTON = "TEST_TAG_SIDEBAR_BUTTON"
const val TEST_TAG_DRAWER_CONTENT = "TEST_TAG_DRAWER_CONTENT"
const val TEST_TAG_DRAWER_CLOSE_BUTTON = "TEST_TAG_DRAWER_CLOSE_BUTTON"
const val TEST_TAG_DRAWER_ITEM_PREFIX = "TEST_TAG_DRAWER_ITEM_"
``` 
Do you think I should still put it in an object ?

Reply:
Having an object doesn't prevent it from being global. I find it better for readability, otherwise, only typing "TEST_TAG" will make Intellisense returns multiple values and make it non-maintainable in the long run. So yeah, I think it would be better

Reply:
Done!